templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/AddCommunityForm.vue

```diff
@@ -156,6 +180,17 @@ export default {
     })
   },
   methods: {
+    getEmojiImg(img) {
+      this.emojiImg = img
+    },
```

Comment:
I believe this is obsolete in which case it should be removed.

Reply:
Removed